elephant/__init__.py

```diff
@@ -19,7 +19,6 @@
                current_source_density,
                change_point_detection,
                phase_analysis,
-               parallel,
```

Comment:
I think it would make sense to add a comment here, explaining why parallel is not autoimported, such that we don't try to 'fix' this in the future.